app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java

```diff
@@ -63,6 +64,7 @@ protected void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
         uri = Uri.parse(getString(R.string.website) + "cgi/user.pl");
+        uri2 = Uri.parse(getString(R.string.website) + "cgi/reset_password.pl");
```

Comment:
For code maintainability, could you change both uri's to reflect what they are actually used for? So for example, uri -> userLoginUri and uri2 -> resetPasswordUri or something similar? This might get confusing in the future, and fixing this now will improve readability. :)